run_fvs.sh

```diff
@@ -6,17 +6,34 @@ date
 pwd
 git status
 
-./build_node.sh
-./create_binary.sh
+docker rm -f host1 || true
+docker run --privileged -v `pwd`:/code --name host1 -tid jpetazzo/dind
+
+docker exec -t host1 bash -c \
+ 'docker -d & \
+ cd /code && \
+ ./build_node.sh && \
+ ./create_binary.sh'
+# docker run --privileged -v `pwd`:/code --name host2 -t jpetazzo/dind bash -c \
+#  'docker -d -D && \
+#  cd /code && \
+#  ./build_node.sh && \
+#  ./create_binary.sh'
+# docker run --privileged -v `pwd`:/code --name host3 -t jpetazzo/dind bash -c \
+#  'docker -d -D && \
+#  cd /code && \
+#  ./build_node.sh && \
+#  ./create_binary.sh'
 
 # Run the FVs
-sudo ./tests/fv/mainline.sh
-sudo ./tests/fv/no_powerstrip.sh
-sudo ./tests/fv/arg_parsing.sh
-sudo ./tests/fv/add_container.sh
-sudo ./tests/fv/unix_socket.sh
-sudo ./tests/fv/add_ip.sh
-sudo ./tests/fv/profile_commands.sh
-sudo ./tests/fv/diags.sh
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/mainline.sh'
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/add_container.sh'
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/add_ip.sh'
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/arg_parsing.sh'
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/profile_commands.sh'
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/no_powerstrip.sh'
+docker exec -t host1 bash -c 'cd /code && sudo ./tests/fv/diags.sh'
+
+docker rm -f host1 || true
```

Comment:
Locally? Just by running `./run_fvs.sh`. I suspect this will change quickly, though. Running multi-host tests takes *forever*, so we'll probably want to have some kind of flag.